### PR TITLE
Validate Contact Form

### PR DESCRIPTION
**Pull request by Gegardus**
In this project, I implemented client-side validation to catch and throw errors in the UI.

### CODE REVIEW

File: form.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  if (emailValid) {
+    msg.remove();
+    form.submit();
+    form.reset();
+  } else {
+    showMessage(input, message, false);
+  }
+});
+
+// Desktop
```
Comment (andersonlebon) [inline form.js:35]: - [ ] Congratulations I can see you are checking the validation before submitting the form but what I would really recommend you to do is to change the way you can display the error to the visitor. One of the requirements says that The error message should show up close to the "submit" button and not overwrite the input as you have done. check the screen video below 👇  👇  on how I am getting your implementation.


https://user-images.githubusercontent.com/65068771/147267295-26c2f822-6d54-4576-8c11-86469dfa6bb2.mp4


- [ ] kindly I would recommend also working on your style by following the design of the template. check the difference here down 👇  👇 

Your design  |  |
-------------|  - |
image |  |
The original design |  |
image |  |

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @Gegardus 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: form.js (hunk 1 above)
Comment (Gegardus) [reply form.js:35]: Hi, Anderson. Thank you for the suggestion. 
But I should ask you to clarify some details. 
In the original design, we have no error message. 
Do you mean text inside textarea? 
If so, would it be good enough to place a message in textarea?
Thank you in advance.

Comment (Gegardus) [reply form.js:35]: Sorry, Anderson, I can’t wait any longer for an answer. My week is in danger of failure. I put the message in the textarea over the button. Hope for a positive answer.

**Review by MrBrN197 — APPROVED** [review]
## Hi, @Gegardus 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gegardus** [comment]
Thank you.

**Pull request merged**